Dataset.py

```diff
@@ -67,6 +67,8 @@ def __init__(self, name=None,
     :param str seq_ordering: "batching"-option in config. e.g. "default", "sorted" or "random".
       See self.get_seq_order_for_epoch() for more details.
     :param int|None partition_epoch:
+    :param int|None repeat_epoch: Repeat the sequences in an epoch this many times. Useful to scale the dataset
+      relative to other datasets, e.g. when used in CombinedDataset.
```

Comment:
How does it work together with `partition_epoch`? Would it repeat that partition? (Extend the documentation to cover this.) Or I guess it doesn't really make much sense to use both options together.
Another thing: Maybe allowing some fraction here, some float, also would make sense? But if you don't need that now, and it complicates other code, then leave it away for now.

Reply:
It is applied after partition_epoch, so yes, you would have a part of the epoch repeated. There is not really a use case for this, I agree. Should I assert that only one is used?

Yes, I even thought, maybe you should be able to set the relative sizes of the datasets in the CombinedDataset. But this requires more code than just using partition and repeat epoch in the sub-datasets. I guess I will wait and see if this is needed in practice.